berserk/clients/tournaments.py

```diff
@@ -24,12 +24,64 @@ def get(self) -> models.CurrentTournaments:
     def get_tournament(self, tournament_id: str, page: int = 1):
         """Get information about a tournament.
 
-        :param tournament_id
+        :param tournament_id: tournament ID
+        :param page: the page number of the player standings to view
         :return: tournament information
         """
         path = f"/api/tournament/{tournament_id}?page={page}"
         return self._r.get(path, converter=models.Tournament.convert)
 
+    def join_arena(
+        self,
+        tournament_id: str,
+        password: str | None = None,
+        team: str | None = None,
+        should_pair_immediately: bool = False,
+    ):
+        """Join a tournament.
+
+        :param tournament_id: tournament ID
+        :param password: tournament password or user-specific entry code generated and shared by the organizer
+        :param team: team with which to join the team battle tournament
+        :param should_pair_immediately: whether to attempt to pair the user if tournament has started
```

Comment:
This seems pretty unclear. I'd maybe just copy the docs: "If the tournament is started, attempt to pair the user, even if they are not connected to the tournament page. This expires after one minute, to avoid pairing a user who is long gone. You may call "join" again to extend the waiting."